x/register/types/events.go

```diff
@@ -16,5 +16,8 @@ const (
 	AttributeKeyCandidateNetworkAddress = "candidate_network_address"
 	AttributeKeyCandidateStatus         = "candidate_status"
 
+	AttributeKeyOZoneLimitIncrease = "ozone_limit_increase"
+	AttributeKeyOZoneLimitDecrease = "ozone_limit_decrease"
```

Comment:
I don't think we need to make the key name to differentiate the increase and decrease. it's just ozone_limit_changes